articles/cognitive-services/Translator/reference/v2-0-reference.md

```diff
@@ -1,804 +1,804 @@
----

```

Comment:
Assuming large diff is from line ending issue

Reply:
I located the update - looks good.